cogs/help.py

```diff
@@ -0,0 +1,167 @@
+        ctx = self.context
+        pages = menus.MenuPages(
+            source=HelpSource(
+                self.get_command_signature,
+                self.filter_commands,
+                ctx.prefix,
+                ctx.author,
+                mapping,
+            ),
+            clear_reactions_after=True,
+        )
+        await pages.start(ctx)
+
+    async def send_cog_help(self, cog: commands.Cog) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=cog.qualified_name,
+            description=(
+                f"Command syntax : `<Those arguments are required>`. `[Those aren't]`\n"
+                f"The prefix for this channel is `{prefix}`\n{cog.description}"
+            ),
+            colour=ctx.bot.colors["blue"],
+        )
+        embed.set_author(name=str(ctx.message.author), icon_url=str(ctx.message.author.avatar_url))
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        for command in await self.filter_commands(cog.get_commands()):
+            embed.add_field(
+                name=f"{prefix}{self.get_command_signature(command)}",
+                value=command.help,
+                inline=False,
+            )
+        embed.set_footer(
+            text=f"Are you interested in {cog.qualified_name} ?",
+            icon_url=str(ctx.bot.user.avatar_url),
+        )
+        await ctx.send(embed=embed)
+
+    async def send_command_help(self, command: commands.Command) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=f"{prefix}{self.get_command_signature(command)}",
+            description=f"Command syntax : `<Those arguments are required>`. `[Those aren't]`\n{command.help}",
+            colour=ctx.bot.colors["blue"],
+        )
+        if command.aliases:
+            embed.add_field(name="Aliases :", value="\n".join(command.aliases))
+        embed.set_author(
+            name=str(ctx.message.author),
+            icon_url=str(ctx.message.author.avatar_url),
+        )
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        if command.hidden:
+            embed.set_footer(
+                text=f"Wow, you found {command.name} !",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        else:
+            embed.set_footer(
+                text=f"Are you interested in {command.name} ?",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        await ctx.send(embed=embed)
+
+    async def send_group_help(self, group: commands.Group) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=f"Help for group {prefix}{self.get_command_signature(group)}",
+            description=(
+                "Command syntax : `<Those arguments are required>`. `[Those aren't]`\n"
+                f"{group.help}"
+            ),
```

Comment:
Same as the suggestion above:
```suggestion
            description=textwrap.dedent(
                f"""
                Help syntax : `<Required argument>`. `[Optional argument]`
                {group.help}
                """
            ),
```